app/src/main/java/org/p2p/wallet/history/interactor/mapper/HistoryTransactionMapper.kt

```diff
@@ -52,8 +52,6 @@ class HistoryTransactionMapper(
         val sourceData = userLocalRepository.findTokenData(finalMintA) ?: return null
         val destinationData = userLocalRepository.findTokenData(finalMintB) ?: return null
 
-        if (sourceData.mintAddress == destinationData.mintAddress) return null
```

Comment:
It was hard to find this mouse, 
cause on account which we used with Nika,
 error transaction was Swap transaction and it had error status. Which was skipped by the parser, and this transactions was not added to history screen

Good bye 3 hours of printing:
[false,false,true,false,false,false,false]
 [false,false,false,false,false,false]

Reply:
But swap from USDC to USDC is impossible.
Something is wrong here.